flexget/utils/tools.py

```diff
@@ -428,20 +428,16 @@ def get_current_flexget_version():
     return flexget.__version__
 
 
-def parse_filesize(text_size, base2=False):
+def parse_filesize(text_size, base2=True):
     """
-    Expands a data size into its bytes as an integer i.e. '1 MB' becomes 1000000.
-    If base2 is specified then assume bibytes so i.e. '1 MB' becomes 1048576
+    Expands a data size into its bibytes as an integer i.e. '1 MB' becomes 1048576.
+    If base2 is false then assume bytes so i.e. '1 MB' becomes 1000000
```

Comment:
Our docstring format goes something like this for parameters:
```rst
:param bool base2: If True, possibly ambiguous units like KB, MB, GB will be assumed to be base 2 units,
    rather than base 10.
```